tests/test_examples.py

```diff
@@ -54,10 +53,7 @@ def init_class_fixtures(cls):
             serialization='pickle',
         )
 
-        copy_market_data(WithTradingEnvironment.MARKET_DATA_DIR,
-                         cls.tmpdir.getpath('example_data/root'))
```

Comment:
Does this mean the tests will query google for the data instead of using the data saved in the repo?